esp/src/src-react/theme-shims/v9ThemeShim.ts

```diff
@@ -34,7 +33,7 @@ const mapAliasColors = (palette: IPalette, inverted: boolean): ColorTokens => {
     colorNeutralForeground3BrandPressed: palette.themeDarkAlt,
     colorNeutralForeground3BrandSelected: palette.themePrimary,
     colorNeutralForeground4: palette.neutralQuaternary,
-    colorNeutralForegroundDisabled: palette.neutralQuaternaryAlt,
+    colorNeutralForegroundDisabled: palette.neutralTertiaryAlt,
```

Comment:
The changes to these colors on line 36 & 112 would undo some changes made in #17743